src/main/res/values/arrays.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string-array name="compress_values">
```

Comment:
Right, I've set ```resources translatable="false"``` but I'm not sure if that works. Google didn't return much information on this:/